README.md

```diff
@@ -14,11 +14,21 @@ The script provides the following capabilities:
 
 Please note that the script is called `aws2-wrap` to show that it works with AWS CLI v2, even though the CLI tool is no longer called `aws2`.
 
-## Install using `pip`
+## Install
+
+### Using `pip`
 
 <https://pypi.org/project/aws2-wrap>
 
-`pip3 install aws2-wrap==1.2.8`
+```
+pip3 install aws2-wrap==1.2.8
```

Comment:
Do we have to include the version here ?

The `pip install` should install the latest version.

Reply:
To be perfectly honest, I can't remember why I did that!